packages/arb-token-bridge-ui/src/token-bridge-sdk/Erc20DepositStarter.ts

```diff
@@ -34,6 +40,44 @@ export class Erc20DepositStarter extends BridgeTransferStarter {
     }
   }
 
+  private getAddress = async (signer: Signer) => {
```

Comment:
These private getters make sure to get the calculated values only ONCE per object, and reuse the values received in previous calls hence avoiding RPC calls.